app/src/main/res/drawable/item_border.xml

```diff
@@ -0,0 +1,12 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid android:color="#FFFFFF" />
+    <stroke
+        android:width="1dp"
+        android:color="#CCCCCC" />
```

Comment:
similar to dimens we want to keep the color hex values also stored in colors.xml and refrence them.

Reply:
As seen in [7a91bc1](https://github.com/priyank1968/CloverEmployeeAppointment/pull/19/commits/7a91bc105cdc8e5076febd0d63dec3df4ab56dd6), refactored the hard-color hex-codes into `color.xml`.